cpp/src/cider/exec/nextgen/jitlib/base/JITValueOperations.h

```diff
@@ -298,6 +254,82 @@ inline JITValuePointer operator>=(T lh, JITValue& rh) {
 inline JITValuePointer operator*(JITValue& value) {
   return value.dereference();
 }
+
+inline JITValuePointer operator&&(JITValue& lh, JITValue& rh) {
```

Comment:
do we need logical `&&`? seems we just use bitwise `&&`.

Reply:
keep consistency with C++ operators.

Reply:
if we only need bitwise operations, we should delete the logical items.

Reply:
We could keep both of them.